src/pages/Search.jsx

```diff
@@ -1,10 +1,11 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import { useSearchParams } from 'react-router-dom';
 import Navbar from '../components/Navbar';
 import MovieGrid from '../components/MovieGrid';
 import LoadingFallback from '../components/LoadingFallback';
 import Pagination from '../components/Pagination';
 import useSearchMovies from '../hooks/useeSearchMovies';
+import useSearchHistory from '../hooks/useeSearchHistory';
```

Comment:
You’re importing from ../hooks/useeSearchMovies and ../hooks/useeSearchHistory.
double “e” in the filenames. fix the file name